kolibri/plugins/device_management/assets/test/views/channel-list-item.spec.js

```diff
@@ -92,8 +92,10 @@ describe('channelListItem', () => {
         const thumb = thumbnail();
         // We are not using the mat-svg webpack loader, so just check for the
         // mat-svg tag here untransformed.
-        expect(thumb.contains('mat-svg')).toEqual(true);
-        expect(thumb.contains('img')).toEqual(false);
+        return wrapper.vm.$nextTick().then(() => {
+          expect(thumb.contains('mat-svg')).toEqual(true);
+          expect(thumb.contains('img')).toEqual(false);
+        });
```

Comment:
@jonboiser not sure why things changed after this PR, but I had to add `nextTick` in a few places to get the tests to pass. Thoughts?